README.md

```diff
@@ -28,4 +28,46 @@ npm install
 
 ```bash
 npm run dev
-```
\ No newline at end of file
+```
+
+6. Navigate to `http://localhost:3000` in your browser to view the UI
+
+## Interacting with the Demo
+
+In your browser you'll see a simple application layout with the option to "Connect Wallet" in the navigation. Follow along with the steps below to interact with the app:
+
+### Creating Capability Object
+
+1. Click the "Connect Wallet" button to connect your wallet. Keep track of which account you use for this portion - we will be using different accounts to read this data. Please ensure you are authenticating with Eth Mainnet for all sign-ins throughout this demo.
+
+2. You will see a secondary signature request with the message "Give this application access to some of your data on Ceramic" - signing this request will create an authenticated browser session that the application will reference when writing data on your behalf. Go ahead and approve this request by clicking "Sign In".
+
+3. Click on the "Write" toggle above the "C1 Endpoint" box. You will be prompted to enter a message - this can be any arbitrary message you'd like to use. Go ahead and enter this here and click "Create".
+
+4. Open up your computer notepad and save the value you'll see under "Stream ID". We will need this later.
+
+5. Grab another Eth address that you own and enter it into the text area underneath "Delegate read access to" - you will see that `did:pkh:eip155:1:` is automatically appended to your address in order to conform to `did:pkh` formatting. Make a note of which address you entered
+
+6. Finally, click "Create Capability" to generate the capability object which delegates read access to the Eth address you own. Paste the result into your notepad as well - we will use this later.
+
+### Reading Data Successfully
+
+1. Manually disconnect the account you used for the previous session and sign in using the address you entered for step 5 from the previous section.
```

Comment:
Or open the app in a new incognito tab

Reply:
Good call - included in [4fc0017](https://github.com/ceramicstudio/private-data-playground/pull/6/commits/4fc0017ccca1cdf4caa07643ddddb4032f8ac908)